docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.8'
+services:
+  postgres:
+    image: 'postgres:13'
+    environment:
+      POSTGRES_PASSWORD: ${DATABASE_PASS}
+      POSTGRES_USER: ${DATABASE_USER}
+      POSTGRES_DB: ${DATABASE_NAME}
+    ports:
+      - "${DATABASE_PORT}:${DATABASE_PORT}"
+    restart: always
+    
```

Comment:
There is no necessity for an blank line. @houssemAmiri

Reply:
line 12 :p

Reply:
docker compose is not finished yet we will have other configurations in the next PR that's why we have blank line more of that the blank line will not change nothing

Reply:
more of that it's a best practise to have blank line in the end of the file

Reply:
mrigel houssem :D